EcoSoapBank/Profile/ProfileDetailViewController.swift

```diff
@@ -12,14 +12,14 @@ class ProfileDetailViewController: UIViewController {
     
     // MARK: - Properties and Outlets
     
-    @IBOutlet weak var avatarImageView: UIImageView!
-    @IBOutlet weak var nameLabel: UILabel!
-    @IBOutlet weak var emailLabel: UILabel!
+    @IBOutlet private weak var avatarImageView: UIImageView!
+    @IBOutlet private weak var nameLabel: UILabel!
+    @IBOutlet private weak var emailLabel: UILabel!
     
-    @IBOutlet weak var editStackView: UIStackView!
-    @IBOutlet weak var nameTextField: UITextField!
-    @IBOutlet weak var emailTextField: UITextField!
-    @IBOutlet weak var avatarURLTextField: UITextField!
+    @IBOutlet private weak var editStackView: UIStackView!
```

Comment:
Private outlets are something I try to do personally as well after Paul Solt pointed out that they can/should be private.